setup.py

```diff
@@ -25,7 +25,7 @@ def read(*parts):
     install_requires=[
         'Django>=2.2.5,<3.1',
         'django-celery',
-        'celery<4.0',
+        'celery<5.0',
```

Comment:
@rudigiesler Do you think it would be worth moving this back to `celery<4.0` for now? Hopefully the new streamlined combinations of builds will all pass for Django 3.1 compatibility.

We can then look into compatibility with `celery<5.0` afterwards.

Reply:
OK, that's been completed.

FYI, the `classifiers` metadata section in `setup.py` will need a few of the `Programming Language...` settings removing.

Also, it looks like this line can go from the `README`:

```md
(r'^djga/', include('google_analytics.urls')),        # This line for Django versions <= 1.11
```

Reply:
Thanks, I've added those two things to the PR as well